packages/protocol/test/governance/validators.ts

```diff
@@ -2182,15 +2182,39 @@ contract('Validators', (accounts: string[]) => {
       await registerValidator(validator)
       await registerValidatorGroup(group)
       await validators.affiliate(group)
-      await mockLockedGold.addSlasher(accounts[3])
     })
 
-    describe('when the sender is one of the whitelisted slashing addresses', () => {
-      it('should succeed', async () => {
+    describe('when the sender is one of three approved contract addresses', () => {
+      beforeEach(async () => {
+        await registry.setAddressFor(CeloContractName.DowntimeSlasher, validator)
```

Comment:
nit: can we use a different address here and explicitly specify the `from` like we do for the others?